Corona-Warn-App/src/androidTest/java/de/rki/coronawarnapp/ui/onboarding/OnboardingDeltaInteroperabilityFragmentTest.kt

```diff
@@ -35,6 +55,15 @@ class OnboardingDeltaInteroperabilityFragmentTest : BaseUITest() {
     fun launch_fragment() {
         launchFragment<OnboardingDeltaInteroperabilityFragment>()
     }
+
+    @Screenshot
+    @Test
+    fun capture_screenshot() {
+        launchFragmentInContainer<OnboardingDeltaInteroperabilityFragment>(themeResId = R.style.AppTheme)
```

Comment:
Should we introduce a parent abstract class that sets this? Or maybe an extension function with default arguments?

When would we want to use a different theme?

Reply:
I introduced  a new inline function `launchFragmentInContainer2` defaults to `AppTheme`. We need to specify the AppTheme here to avoid showing: 1- ActionBar 2- Squeezing some UI elements. I mentioned also this in the docs for the function mentioned above